Optimize a few things

thetwopct opened the pull request:
Realised this repo could do with some love 😢 ❤️ 

1. Alphabetizised the projects in the readme and in each level index.
2. Moved projects between levels indexes where they have progressed.
3. Kube OVN listing was missing so has been re-added.
4. Fluentbit is not a project and removed from Graduated level index (sub-project of fluent)
5. Updated some file names to be consistent (will be continued after a-z is merged)
6. Ran some SVGs through autocrop as formatting was off (will be continued after a-z is merged)

cc @cjyabraham

cjyabraham commented on README.md:
```diff
@@ -7,167 +7,166 @@ With many browsers, you can right click and copy the logo of your choice below.
+  * [Konveyor](examples/sandbox.md#konveyor-logos)
+  * [Krator](examples/sandbox.md#krator-logos)
   * [Krustlet](examples/sandbox.md#ckrustlet-logos)
+  * ~~[Kube-OVN](examples/sandbox.md#kube-ovn-logos)~~
```
Why is this strikethrough?

thetwopct replied:
It was missing, but I re-added it.

cjyabraham reviewed (changes requested):
Looks like Archived Projects listing in the README is missing [a few](https://www.cncf.io/archived-projects/)?

For the white logos, could we use a darker grey background to help them stand out more?

The [Akri stacked logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#akri-logos) are broken.

Why are the [Devfile logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#devfile-logos) so small?

Can you make sure the white logos go on a grey background? [These ones are problematic](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#merbridge-logos) and there are several others. Maybe we should just use a grey background throughout to keep it simpler?

[Open Service Mesh logos are broken](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#open-service-mesh-logos).

[Service Mesh Interface Logos](https://github.com/thetwopct/artwork/blob/optimize/examples/sandbox.md#service-mesh-interface-logos) are broken.

thetwopct commented:
@cjyabraham 

1) Out of scope. I didn't compare data on that level; if the logos are missing, the logos likely don't exist in the repo. As I work through the whole repo this will become clearer. This is part 1 of possibly many parts. Wasn't expecting to need to completely rejig the MD files, but it's essential to do that first because otherwise any other PRs will have issues merging. Welcome to compare and PR if needed.
2) Out of scope. Thats not something I will be covering in this optimisation. But the MD files don't support `<style>` tag, so it would need to be inline styles on every table. PRs welcome.
3) Done. Had renamed them and missed "stack" changing to "stacked"
4) Done. This PR only includes updates to about 5 folders of images out of about 250 folder updates required. But have fixed this for you by running it through Autocrop now instead of later.
5) On the list. Merbridge goes outside the conventions of color + black + white layout of all other logos so this logo pack needs to be updated and the table needs to be updated. There are maybe 5-10 projects that need fixing in this way.
6) Done. It's been archived. Removed and updated archived.md
7) Done. It's been archived. Removed and updated archived.md

edsiper commented:
@thetwopct why Fluent Bit has been removed without asking Fluent maintainers about this ?

Fluent Bit is a graduated project under the umbrella of Fluentd, this has been approved while ago by CNCF.

thetwopct commented:
@edsiper ok. Is there a reason why the project is not listed on the Projects page under Graduated? 

https://www.cncf.io/projects/

edsiper commented:
sub projects are not listed in the principal page

thetwopct commented:
@edsiper Hey Eduardo, we have [restored the fluentbit artwork to the repo](https://github.com/cncf/artwork/blob/main/examples/graduated.md#fluentbit-logos) and improved how we handle the display of subprojects so hopefully this should be clear going forward. Thanks for bringing this to our attention.

edsiper commented:
@thetwopct thank you !
